rename_imports.py

```diff
@@ -0,0 +1,78 @@
+
+from LSP.plugin.core.file_watcher import FileWatcherEvent, FileWatcherProtocol, get_file_watcher_implementation, FileWatcher
+from LSP.plugin.core.typing import  List
+import os
+import sublime
+
+
+def register_rename_import_file_watcher(root_path: str):
+    file_watcher = get_file_watcher_implementation()
+    if file_watcher:
+        watcher = file_watcher.create(
+            root_path=root_path,
+            patterns=["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"],
+            events=["create", "delete"],
+            ignores=['**/node_modules/**'],
+            handler=HandleRenameImport
+        )
+
+
+queued_events = []  # type: List[FileWatcherEvent]
```

Comment:
Events should be bound to specific server instance, not global. Too much room for error otherwise.

Reply:
moved to property in a class.